app/components/avo/actions_component.rb

```diff
@@ -61,8 +71,8 @@ def on_index_page?
     !on_record_page?
   end
 
-  def icon(action)
-    svg action.icon, class: "h-5 shrink-0 mr-1 inline pointer-events-none"
+  def icon(icon)
+    svg icon, class: "h-5 shrink-0 mr-1 inline pointer-events-none"
   end
 
   def render_item(action)
```

Comment:
I think this whole method needs a refactor at some point.

Reply:
I done that in [this commit](https://github.com/avo-hq/avo/pull/3218/commits/04716bf3020c509b98f6f500b2047fd47f1a72eb) than reverted it =))